### ISSUE TITLE
get data fails

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ndio#115]
| ndio version | os | python |
| --- | --- | --- |
| 0.1.11 | macosx | 2.7 |

This command fails:

```
token = bock11
channel = image
res = 1
imout = oo.get_volume(token, channel, 0, 1800, 0, 1800, 2917, 3400, resolution=res, memory=True)

```

With the following message:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-14-89ce2579f38c> in <module>()
----> 1 imout = oo.get_volume(token, channel, 0, 1800, 0, 1800, 2917, 3400, resolution=res, memory=True)
      2 

/Users/graywr1/code/ndio/ndio/remote/neurodata.pyc in wrapped(self, *args, **kwargs)
     67                 if self.ping('{}/info/'.format(token)) != 200:
     68                     raise RemoteDataNotFoundError("Bad token {}".format(token))
---> 69             return f(self, *args, **kwargs)
     70         return wrapped
     71 

TypeError: get_volume() got an unexpected keyword argument 'memory'
```

Seems like something simple that's fundamentally broken (for me)

**Comment by j6k4m8** [comment]
don't pass `memory` as an arg — it now automagically does the smartest thing, based on what version of ndio you're using and what version of python you're using :)

**Comment by j6k4m8** [comment]
fixed in #116 :)
